packages/accounts/src/msca/plugins/multi-owner/extension.ts

```diff
@@ -19,11 +19,25 @@ export type MultiOwnerPluginActions<
   readOwners: (
     params: { pluginAddress?: Address } & GetAccountParameter<TAccount>
   ) => Promise<ReadonlyArray<Address>>;
+
+  isOwnerOf: (
+    params: {
+      address: Address;
+      pluginAddress?: Address;
+    } & GetAccountParameter<TAccount>
+  ) => Promise<boolean>;
 } & (IsUndefined<TAccount> extends false
     ? {
         readOwners: (
           params?: { pluginAddress?: Address } & GetAccountParameter<TAccount>
         ) => Promise<ReadonlyArray<Address>>;
+
+        isOwnerOf: (
+          params: {
+            address: Address;
+            pluginAddress?: Address;
+          } & GetAccountParameter<TAccount>
+        ) => Promise<boolean>;
```

Comment:
this isn't needed since you will always have at least 1 parameter to this call.

The goal of this additional section here is for cases where all of the parameters become optional. In that case, not having this means calls for eg look like `readOwners({})` when the client is connected. for `isOwnerOf` the call will always be at least `isOwnerOf({ address: "0xaddress" })`

Reply:
```suggestion
```

Reply:
isOwnerOf is not currently part of the multi owner plugin actions right? so I added it. Right now, you need to interact directly with the plugin to call methods not in the plugin manifest?

Reply:
sorry what I mean is that you have it defined twice. Once here: https://github.com/alchemyplatform/aa-sdk/pull/439/files#diff-65e2e6f935fcbe2e2228c6abe16d2e64472fdf44c8c57c68ce5bdfd2e14f9c5aR23

and once on this line too

the purpose of the typing following https://github.com/alchemyplatform/aa-sdk/pull/439/files#diff-65e2e6f935fcbe2e2228c6abe16d2e64472fdf44c8c57c68ce5bdfd2e14f9c5aR29 is to allow for us to define `params: ...` as `params?: ...` when all of the params are optional. for `isOwnerOf` the `address` param is always required so there's no need to add it here

Reply:
for example:
```
readOwners: (
    params: { pluginAddress?: Address } & GetAccountParameter<TAccount>
  ) => Promise<ReadonlyArray<Address>>;
```
only has one parameter in addition to the `account` param, but that param is optional. Depending on the type of `TAccount` then that makes `account` optional sometimes. So if `account` and `pluginAddress` are both optional, then the type for `readOwners` should just be:

```
readOwners: (
    params?: { pluginAddress?: Address } & GetAccountParameter<TAccount>
  ) => Promise<ReadonlyArray<Address>>;
```
with `params?`, otherwise you have to call it as `readOwners({})` which isn't a great API. So this section is used to define the methods that can be called as `readOwners()` when the client has the account hoisted